tests/nativeapp/test_artifacts.py

```diff
@@ -1161,15 +1161,6 @@ def test_symlink_or_copy_with_no_symlinks_in_project_root(snapshot):
                     dst=Path(deploy_root, "Grand4/Parent3"),
                     deploy_root=deploy_root,
                 )
-            assert not Path(deploy_root, "Grand4/Parent3").is_symlink()
-            assert not Path(deploy_root, "Grand4/Parent3/ChildA").is_symlink()
-            assert Path(deploy_root, "Grand4/Parent3/ChildB.py").is_symlink()
-            assert Path(deploy_root, "Grand4/Parent3/ChildC").is_symlink()
-            assert not Path(
-                deploy_root, "Grand4/Parent3/ChildC/GrandChildA"
-            ).is_symlink()
-
-            assert_dir_snapshot(Path("./output/deploy/Grand4/Parent3"), snapshot)
```

Comment:
The lines above are being removed because:
1. The main thing to check is that we received a NotInDeployRootError
2. Even though an exception was raised, there might have been side effects to the filesystem already. Without `sorted`, these side effects can occur in random order, and hence checks from 1164 to 1172 are not guaranteed to always work. 
3. Even with sorted, these checks do not offer additional benefit, because the exception has already been verified. Realistically, a user would run `bundle` again and it will delete the deploy root and create it from scratch again.